demo/api/src/db/migrations/Migration20230227072819.ts

```diff
@@ -0,0 +1,13 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20230227072819 extends Migration {
```

Comment:
This migration should be in the library. Furthermore, updatedAt should be set based on the current attached document imo